### DISCUSSION TITLE
Is there a setting for text wrapping inside a shape?

### DISCUSSION THREAD

**Discussion by Fierce-Ninja** [zsviczian/obsidian-excalidraw-plugin#171] [Q&A]
Hi sorry if this is a silly question. Is there a setting that would allow me to make sure the text gets wrapped when typing text inside a shape?

**Comment by YesterKo** [comment]
Unfortunately, it's not yet possible. You can, however, transclude text from another note and you can then define how many characters per line you want, with `![[document#^blockref]]{40}`
For more info, see #118 and excalidraw/excalidraw#1428.
